### ISSUE TITLE
Problem with displaying customized "Submit for Review" pop-up with specific size by default

### ISSUE THREAD

**Issue by BPeuch** [IQSS/dataverse#6900]
Version: 4.20

Hello everybody,

We have customized the pop-up window that shows up when, on a dataset's webpage, a user clicks on ```Submit for Review```.

We have changed the text (with message classes) as well as the label of the ```Submit``` button and we added a PDF viewer that contains a deposit agreement which the user must read and approve to submit their dataset for publication:

![new_submit_window](https://user-images.githubusercontent.com/15209224/81424232-ea74eb00-9155-11ea-8925-5371545ee746.png)
This is almost perfect, but we wish the window would automatically display in a less "squashed" manner, i.e. as a "standing" rectangle instead of a "lying" rectangle as it currently does. As it stands, users will necessarily have to resize the window to make the text display in a more reading-friendly resolution.

(We know that users can resize the window by themselves, and in two ways, even: both by clicking and dragging the edges and the corners of the window, and by clicking the arrow/"Expand" button in the top right corner. But we would rather they did not have to do it.)

The window behaves like this regardless of the Web browser.

Here's how this was technically achieved:
- We set both the width and height of the dialog box at 100%:
```<p:dialog id="inreview" header="#{bundle['dataset.submitBtn']}" widgetVar="inreview"  modal="true" width="100%" height="100%"```
- To add the PDF viewer to make the deposit agreement easier to read, we inserted an iFrame:
```<IFRAME src="http://dataversetest.arch.local/termsSoda/inreview.pdf" width="90%" height="90%" frameborder="0" ></IFRAME>```
- Finally, we added this line to the CSS file in order to make the "Accept and Continue" (formerly "Submit") and "Cancel" buttons always visible (otherwise, they disappear underneath the bottom limit of the pop-up window, and the user has to scroll down to see them):
```.ui-dialog .ui-dialog-content {overflow: visible; }```

Following @TaniaSchlatter's advice, I'm tagging her and @mheppler since this issue concerns UI, UX and front-end aspects.

Do you see how we could make the window automically display as desired?

**Comment by mheppler** [comment]
At first glance, I don't see anything that would cause the height issue you are seeing.

One question, have you tried the [PrimeFaces multimedia component](https://www.primefaces.org/showcase/ui/multimedia/media.xhtml)?

**Comment by BPeuch** [comment]
Hello @mheppler. Thanks for your tip, and sorry for this very late reaction.

We looked into the PrimeFaces documentation but the problem proved rather tricky to solve. In the end however, we settled for the default display, judging that, after all, users can easily resize the window themselves + download the PDF to read it as their leisure.

Feel free to close this issue. (I see that I can do it by myself but I don't know if that's standard procedure?)
